ledger/error/src/main/scala/com/daml/error/ErrorCategory.scala

```diff
@@ -332,8 +333,28 @@ object ErrorCategory {
       )
       with ErrorCategory
 
-  implicit val orderingErrorType: Ordering[ErrorCategory] = Ordering.by[ErrorCategory, Int](_.rank)
+  @Description(
+    """This error category is used to signal that an unimplemented code-path has been triggered by a client or participant operator request.
+      |Such an error can arise expectedly if the operator submits an admin request against a participant that does not, partly or fully,
+      |support the targeted feature. This error can also arise when a user request ends up leveraging, unexpectedly, an unsupported feature."""
```

Comment:
```suggestion
```
I'd be inclined to remove this explanation. It's a bit fuzzy; and I don't know how to make it better. I believe the most important information is the resolution below.